recipe/meta.yaml

```diff
@@ -19,9 +19,11 @@ source:
     # - https://bugs.python.org/issue28708
     # - https://github.com/zeromq/libzmq/issues/1165
     - fd_setsize.patch                # [win]
+    - mingw-w64--DMS_WINXX.patch
+    - win-msc_ver-1900.patch
 
 build:
-  number: 1
+  number: 2
```

Comment:
Thanks @isuruf! That explains the `rpy2` problem. I'll send a PR to bump it now.